Makefile

```diff
@@ -0,0 +1,27 @@
+MAINTAINER = "CS50 <sysadmins@cs50.harvard.edu>"
+NAME = adminer50
+VERSION = 1.0.0
+
+.PHONY: clean
+clean:
+	rm -rf opt $(NAME)_$(VERSION)_*.deb
+
+.PHONY: deb
+deb: clean
+	mkdir -p opt/cs50/adminer
+	cp -r adminer-4.3.1.php  bin  css  fonts  images  index.php  Makefile  plugins opt/cs50/adminer
```

Comment:
No need for double spaces?

Reply:
Oops, copied from the output of `ls`. Fixed.